govcd/api_vcd_unit_test.go

```diff
@@ -0,0 +1,77 @@
+// +build api unit ALL
```

Comment:
As discussed in chat, this code would be better placed inside `api_vcd_test.go`

Reply:
Moved.